pinot-segment-local/src/main/java/org/apache/pinot/segment/local/indexsegment/mutable/MutableSegmentImpl.java

```diff
@@ -242,6 +235,10 @@ public long getLatestIngestionTimestamp() {
     // Initialize for each column
     for (FieldSpec fieldSpec : _physicalFieldSpecs) {
       String column = fieldSpec.getName();
+      boolean isRaw = isNoDictionaryColumn(noDictionaryColumns, invertedIndexColumns, fieldSpec, column);
```

Comment:
If you rename `isRaw` to `isNoDictionary`, you don't need to negate it in both places it's being used and IMO it reads better.

Reply:
Changing the name doesn't change its truth value?!

Reply:
Changed it to `isDictionary` and inverted its truth value here.

Reply:
You can go either way, but IMO they're essentially the same. The method name is `isNoDictionaryColumn()`, so it's more consistent with that too.